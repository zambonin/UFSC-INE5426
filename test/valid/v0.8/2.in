float values[100]
values[([int] values[10])+2]= - values[([int] values[12])*5]

