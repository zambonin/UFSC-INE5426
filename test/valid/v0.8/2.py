exec(open('../../../src/scope_manager.py', 'r').read())
values = [0] * 100
values[(int(values[10]) + 2)] = -values[(int(values[12]) * 5)]
