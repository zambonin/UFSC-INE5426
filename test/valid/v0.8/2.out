float array: values (size: 100)
= [index] values + [int] [index] values 10 2 -u [index] values * [int] [index] values 12 5
